DiscordLink/DiscordLink.csproj

```diff
@@ -78,10 +79,17 @@
       <HintPath Condition="Exists('..\..\Eco\Server\Eco.Simulation\bin\Release\net7.0-windows\Eco.Simulation.dll')">..\..\Eco\Server\Eco.Simulation\bin\Release\net7.0-windows\Eco.Simulation.dll</HintPath>
       <Private>false</Private>
     </Reference>
-    <Reference Include="Eco.WorldGenerator">
-      <HintPath Condition="Exists('..\Dependencies\Eco.WorldGenerator.dll')">..\Dependencies\Eco.WorldGenerator.dll</HintPath>
-      <HintPath Condition="Exists('..\..\Eco\Server\Eco.WorldGenerator\bin\Release\net7.0-windows\Eco.WorldGenerator.dll')">..\..\Eco\Server\Eco.WorldGenerator\bin\Release\net7.0-windows\Eco.WorldGenerator.dll</HintPath>
-    </Reference>
+	<Reference Include="Eco.WorldGenerator">
+	  <HintPath Condition="Exists('..\Dependencies\Eco.WorldGenerator.dll')">..\Dependencies\Eco.WorldGenerator.dll</HintPath>
+	  <HintPath Condition="Exists('..\..\Eco\Server\Eco.WorldGenerator\bin\Release\net7.0-windows\Eco.WorldGenerator.dll')">..\..\Eco\Server\Eco.WorldGenerator\bin\Release\net7.0-windows\Eco.WorldGenerator.dll</HintPath>
+	  <Private>false</Private>
+	</Reference>
+	<Reference Include="Eco.WebServer">
+	  <HintPath Condition="Exists('..\Dependencies\Eco.WebServer.dll')">..\Dependencies\Eco.WebServer.dll</HintPath>
+	  <HintPath Condition="Exists('..\..\Eco\Server\Eco.WebServer\bin\Release\net7.0-windows\Eco.WebServer.dll')">..\..\Eco\Server\Eco.WebServer\bin\Release\net7.0-windows\Eco.WebServer.dll</HintPath>
+	 <Private>false</Private>
+	</Reference>
```

Comment:
I cannot for the life of me seem to be able to fix the indenting. Tabs keep being inserted.